client/src/game/managers/AOInputProcessor.java

```diff
@@ -208,9 +235,157 @@ private void doAlternativeActions(int keycode) {
                 // Toggle between Windowed Mode and Fullscreen.
                 gui.toggleFullscreen();
                 break;
+            //asd
+            case Input.Keys.NUM_1:
+                useq1();
+                break;
+            case Input.Keys.NUM_2:
+                useq2();
+                break;
+            case Input.Keys.NUM_3:
+                useq3();
+                break;
+            case Input.Keys.NUM_4:
+                useq4();
+                break;
+            case Input.Keys.NUM_5:
+                useq5();
+                break;
+            case Input.Keys.NUM_6:
+                useq6();
+                break;
+            case Input.Keys.Q:
+                asigQI ();
+                break;
+
+//asdf
+        }
+    }
+//asd
```

Comment:
aca tmb

Reply:
oks para la próxima tendré cuidado de subir sin todos los marcadores e codigo innecesario.

Reply:
Barbaro, igualmente siguen habiendo un monton de comentarios `asd` y codigo sin usar / comentado en tu PR

Habria que limpiarlo todo

Reply:
> Hola @bestiardo, gracias por el apoyo, excelente aporte, teníamos pensado meter mano en eso! #106 Ahí testie el pr.
> Antes de comentarte las cosas note. Podríamos conversar para comprender bien el concepto de usabilidad que quisiste implementar? Estas en el discord? Mi usuario es Derillo#3967
> Abrazo!

De momento no tengo discord cuando tenga algo de tiempo me lo descargo y veo como se usa.
Y lo del inventario lo hice mas por que me gusta la idea de ver todo el inventario en un juego de rol. Por curiosidad y probar si con mis conocimientos de programar que no  llegan a lo mas básico, y por lejos de llagar, podía hacerlo.
la verdad que el del gif se ve bastante bueno pero como lo que baje solo tenia ese inventario que se scroleava y no te permitia usar los items mas alla del 6to pense meterle mano.

Reply:
> aca tmb

una pregunta soy nuevo usando básicamente todo esto en especial el github después de actualizar todos los archivos en mi fork del repositorio tengo que hacer un nuevo commit de los archivos que les quite el código basura que había dejado?

Reply:
> > Hola @bestiardo, gracias por el apoyo, excelente aporte, teníamos pensado meter mano en eso! #106 Ahí testie el pr.
> > Antes de comentarte las cosas note. Podríamos conversar para comprender bien el concepto de usabilidad que quisiste implementar? Estas en el discord? Mi usuario es Derillo#3967
> > Abrazo!
> 
> De momento no tengo discord cuando tenga algo de tiempo me lo descargo y veo como se usa.
> Y lo del inventario lo hice mas por que me gusta la idea de ver todo el inventario en un juego de rol. Por curiosidad y probar si con mis conocimientos de programar que no llegan a lo mas básico, y por lejos de llagar, podía hacerlo.
> la verdad que el del gif se ve bastante bueno pero como lo que baje solo tenia ese inventario que se scroleava y no te permitia usar los items mas alla del 6to pense meterle mano.

Bueno despreocupate por lo del discord, si quieres escribime al correo donderillo@gmail.com y lo vemos.
Es maravilloso que te hayas animado y te hayas tomado el tiempo, se agradece.
Respecto a los comentarios que te hace @RecoX despreocupate que son cambios mínimos.

Lo importante definir sería que:
el funcionamiento de la Q debería ser al revés, lo que se abre es el inventario para asignar con la funcionalidad que implementaste de SHIFT+NUM al QuickBar.

Entre otras cosas noté que ahora ya no se puede scrollear el QuickBar, eso deberíamos mantenerlo.
Si te animás hay más cuestiones que queremos implementar para el QuickBar, así que si seguís entusiasmado, podemos ir viéndolo paso a paso jaja xD.
También te recomiendo pasar por este issue #196 
Abrazo y seguimos en contacto!

Reply:
> > Hola @bestiardo, gracias por el apoyo, excelente aporte, teníamos pensado meter mano en eso! #106 Ahí testie el pr.
> > Antes de comentarte las cosas note. Podríamos conversar para comprender bien el concepto de usabilidad que quisiste implementar? Estas en el discord? Mi usuario es Derillo#3967
> > Abrazo!
> 
> De momento no tengo discord cuando tenga algo de tiempo me lo descargo y veo como se usa.
> Y lo del inventario lo hice mas por que me gusta la idea de ver todo el inventario en un juego de rol. Por curiosidad y probar si con mis conocimientos de programar que no llegan a lo mas básico, y por lejos de llagar, podía hacerlo.
> la verdad que el del gif se ve bastante bueno pero como lo que baje solo tenia ese inventario que se scroleava y no te permitia usar los items mas alla del 6to pense meterle mano.

Hola @bestiardo , te dejo el [link](https://discord.gg/ytk8Pcc) del nuestro servidor de Discord. para cuando te lo bajes

Reply:
link al servidor de Discord me manda a 1 pargina de error. invitacion expirada o no valida o puede que no tengas permiso de unirte

Reply:
http://www.argentumonline.org